tests/rest/test-rest-api.php

```diff
@@ -276,6 +424,25 @@ public function test_rest_api_returns_error_for_classic_content() {
 		wp_delete_post( $post_id );
 	}
 
+	public function test_rest_api_returns_error_for_include_and_exclude_filter() {
+		$post_id = $this->get_post_id_with_content( '<p>Classic editor content</p>' );
```

Comment:
We should ensure the post content is valid Gutenberg blocks, or it'll might hit the classic content error from the test above.

```suggestion
		$post_id = $this->get_post_id_with_content( '<!-- wp:paragraph --><p>content</p><!-- /wp:paragraph -->' );
```

Reply:
Merge conflict side effect unfortunately as I had the Gutenberg blocks in there previously. Good catch